### PR TITLE
Troubleshooting: my dashboard is slow

### PR DESCRIPTION
**Pull request by gvwilson**
A troubleshooting guide that starts with a question.

### CODE REVIEW

File: docs/troubleshooting-guide/my-dashboard-is-slow.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## Is Metabase and/or the data warehouse running on under-powered hardware?
+
+**Root cause:** If you run Metabase or the underlying data warehouse on a ten-year-old machine with only 1 GByte of RAM, it may not be able to keep up with your demands.
+
+**Steps to take:**
+
+1. Check the performance logs for the server where the database is running to see if it is hitting CPU or memory limits.
+2. Check the performance logs for the server where Metabase is running.
+3. If either is a bottleneck, upgrade to a more powerful server or one with more memory. If you would like this taken care of you, along with backups, cache configuration, and so on, please consider using [Metabase Cloud][metabase-start].
```
Comment (salsakran) [inline docs/troubleshooting-guide/my-dashboard-is-slow.md:98]: I'd skip the CTA there. Let's keep the troubleshooting guide strictly a helpful resource. Also remember that people on cloud will use this (ideally as their first step)

File: docs/troubleshooting-guide/my-dashboard-is-slow.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
```
Comment (salsakran) [inline docs/troubleshooting-guide/my-dashboard-is-slow.md:99]: one additional one to consider is for cases where a dashboard starts to slow down
1. did data volume grow? -> throw more hardware at it
2. are there more users on the data warehouse -> provision or set up quotas
3. are your execution slots in a database being saturated -> increase available execution slots

**Review by salsakran — COMMENTED** [review]
minor nits, but a huge improvement. 👍

File: docs/troubleshooting-guide/my-dashboard-is-slow.md (hunk 2 above)
Comment (paoliniluis) [reply docs/troubleshooting-guide/my-dashboard-is-slow.md:99]: agree with 1, but for 2 and 3 I would include something like: "ask your infrastructure team if there is any enqueuing being performed at the DH, if there is, ask them to raise the priority of Metabase queries", to make it more general for all the data sources we support

**Review by dragonsahead — APPROVED** [review]

File: docs/troubleshooting-guide/my-dashboard-is-slow.md (hunk 1 above)
Comment (gvwilson) [reply docs/troubleshooting-guide/my-dashboard-is-slow.md:98]: Removed.

File: docs/troubleshooting-guide/my-dashboard-is-slow.md (hunk 2 above)
Comment (gvwilson) [reply docs/troubleshooting-guide/my-dashboard-is-slow.md:99]: New section added at the top.

**Review by dragonsahead — APPROVED** [review]
ultra mega approved :D

**Pull request closed without merging**